lib/liquid/standardfilters.rb

```diff
@@ -355,6 +355,16 @@ def floor(input)
       raise Liquid::FloatDomainError, e.message
     end
 
+    def max(input, n)
+      result = [input, n].map { |number| Utils.to_number(number) }.min
```

Comment:
Allocating an array just for this is not great, we're not running on something fancy like [TruffleRuby](https://github.com/graalvm/truffleruby) that can optimize that allocation away.